api/swagger.yaml

```diff
@@ -3429,6 +3456,14 @@ components:
               allOf:
                 - description: Minimum weight allowed
                 - $ref: '#/components/schemas/valueWithUnit'
+          example:
+            weightConstraintType: range
```

Comment:
Zou het niet mooi zijn om niet elke keer een `range` voorbeeld te doen, maar bijvoorbeeld ook `maximum` en `minimum`, zodat je -verspreid over verschillende voorbeelden- zoveel mogelijk mogelijikheden laat zien?

Reply:
Dat is een goed idee, ik koos hier bewust voor `range` omdat het alle mogelijke velden bevat.